PocketKit/Sources/PocketKit/Main/MainViewModel.swift

```diff
@@ -1,59 +1,123 @@
 import Combine
+import Network
 import Sync
 import Foundation
 import BackgroundTasks
 import UIKit
+import Textile
 
 @MainActor
 class MainViewModel: ObservableObject {
+    let home: HomeViewModel
+    let saves: SavesContainerViewModel
+    let account: AccountViewModel
+    let source: Source
+
     @Published
     var selectedSection: AppSection = .home
 
     @Published
-    var isCollapsed = UIDevice.current.userInterfaceIdiom == .phone
+    var bannerViewModel: PasteBoardModifier.PasteBoardData?
 
-    let home: HomeViewModel
-    let saves: SavesContainerViewModel
-    let account: AccountViewModel
+    @Published
+    var showBanner: Bool = false
+
+    private var subscriptions: Set<AnyCancellable> = []
+
+    @MainActor
```

Comment:
because the class itself is marked with `@Mainactor`, this should be redundant here